articles/install-guide/pyinstall.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Develop with Q# + Python
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.python
+---
+
+# Develop with Q# + Python
+
+This is the installation guide for the QDK environment to program hybrid Q# and Python programs.
+
+1. Pre-requisites
+
+    - [Python](https://www.python.org/downloads/) 3.6 or later
+    - The [PIP](https://pip.pypa.io/en/stable/installing) Python package manager
+    - [.NET Core SDK 3.0 or later](https://www.microsoft.com/net/download)
+
+
+1. Install the `qsharp` package, a Python package that allows the interoperation between Q# and Python.
+
+    ```bash
+    pip install qsharp
+    ```
+
+1. Install the `iqsharp`, an extension primarily used by Jupyter and Python that provides the core functionality for compiling and simulating Q# operations.
+
+    ```bash
+    dotnet tool install -g Microsoft.Quantum.IQSharp
+    dotnet iqsharp install
+    ```
+  
+1. Although it's not strictly necessary, we highly recommend to use the lightweight Visual Studio Code (VS Code) IDE to edit the Q# and Python files. Together with the QDK extension for VS Code this will provide code completion and syntax syntax highlighting features to make the coding easier.
```

Comment:
This may oversell things a bit, in that the VS Code extension offers comparatively limited functionality when editing Q# code called from Python, due to the language server not getting the information it needs about build-time dependencies (microsoft/iqsharp#24). Thus, I would suggest that we recommend VS Code for use with Python and Q#, but not get into specific features until we have a plan for how to support those features.

Reply:
Ok, it makes sense. Although the edits of the installation guide file are from another PR (#616) which in theory was ready for publishing, but things in December go slow. I'll fix that there. 

Those changes get listed here because I used the same fork for both PRs (Git Hub noob mistake).